src/frontend/src/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,23 @@
+import { ThreeCircles } from  'react-loader-spinner'
+
+export default function LoadingSpinner() {
+    return (
+        <div className={`loading-spinner`} >
+            <ThreeCircles
+            height="150"
+            width="150"
+            color=""
+            wrapperStyle={{
+                "margin": "auto",
+                "padding":"15%"
+            }}
+            wrapperClass="loading-spinner circles"
+            visible={true}
+            ariaLabel="three-circles-rotating"
```

Comment:
I'd assume visible is `true` by default, can we remove it? Also let's skip aria labels, as we aren't targeting accessibility for the MVP


```suggestion
```